dnf5/main.cpp

```diff
@@ -1393,14 +1393,14 @@ int main(int argc, char * argv[]) try {
 
                 if (auto transaction_store_path = context.get_transaction_store_path();
                     !transaction_store_path.empty()) {
-                    std::cout << "The operation will only store the transaction in " << transaction_store_path << "."
+                    std::cerr << "The operation will only store the transaction in " << transaction_store_path << "."
```

Comment:
It'd be great if, in places where we have access to the context class with custom error or output stream instances set up by the user, we use those.

Reply:
Agreed, though there are many uses of `std::cout` and `std::cerr` in the codebase. I changed the ones in `main.cpp` and `context.cpp` to use `Context::print_error` and `Context::print_output`. Many of the other uses are in DNF 5 commands, so they're not likely to be called by API consumers and are probably a low priority to change.